kong/plugins/datadog/handler.lua

```diff
@@ -146,7 +143,8 @@ local function log(premature, conf, message)
 
       logger:send_statsd(stat_name, stat_value,
                          logger.stat_types[metric_config.stat_type],
-                         metric_config.sample_rate, metric_config.tags)
+                         metric_config.sample_rate, 
+                         {"app:kong", "api_name:" .. name})
```

Comment:
ditto, `api_name` is incorrect in the context of Services/Routes

Reply:
Recommendations...?  "route"?  "service"?  What's more appropriate?